bflib/src/db/group.rs

```diff
@@ -1103,6 +1103,27 @@ impl Db {
         artillery
     }
 
+    pub fn alcm_near_point(&self, side: Side, pos: Vector2) -> SmallVec<[GroupId; 8]> {
+        let range2 = (self.ephemeral.cfg.alcm_mission_range as f64).powi(2);
+        let alcm = self
+            .deployed()
```

Comment:
just change this to actions() (or if that function doesn't exist you could easily create it)